modules/Search/components/Facet.module.scss

```diff
@@ -1,29 +1,3 @@
-.container {
```

Comment:
_Comment placed in an unrelated place_

There is weird content shift when clicking some facets on desktop. Let me know if you want to fix it in this PR, or file a new ticket for it.

https://user-images.githubusercontent.com/1641218/185590657-634d09ed-37d1-4bbb-8917-7abde8ed382b.mov